Source/WPF/MyMoney/Database/SqlDatabase.cs

```diff
@@ -2201,6 +2211,7 @@ public void UpdateCurrencies(Currencies currencies)
                     sb.Append(string.Format(",'{0}'", DBString(s.Symbol)));
                     sb.Append(string.Format(",{0}", s.Ratio));
                     sb.Append(string.Format(",{0}", s.LastRatio));
+                    sb.Append(string.Format(",{0}", DBString(s.CultureCode)));

```

Comment:
This `{0}` needs to be surrounded by single quotes because it a string type.

Reply:
fixing this now

Reply:
fixed